### PR TITLE
Drop create link / paste as symlink functionality

### PR DESCRIPTION
**Pull request by jelly**
This feature has plenty of issues and needs to be reworked to be a lot more simple. Either it will just create a link like GNOME Files as absolute link or relative link which would require more thought when moving.

### CODE REVIEW

**Comment by martinpitt** [comment]
Argh, `li:nth-child(4)` tests - maybe better change that test to not assume particular indexes? (But also ok to just fix it)

File: src/fileActions.jsx [hunk 1]
```diff
@@ -546,49 +473,21 @@ export const fileActions = (path, files, selected, setSelected, clipboard, setCl
...
-                targetPath
-            ]).catch(err => addAlert(err.message, "danger", new Date().getTime()));
-        }
+    const spawnPaste = (sourcePath, targetPath) => {
+        cockpit.spawn([
+            "cp",
+            "-R",
+            ...sourcePath,
+            targetPath
+        ]).catch(err => addAlert(err.message, "danger", new Date().getTime()));
```
Comment (cockpituous) [inline src/fileActions.jsx:482]: These 7 added lines are not executed by any test.

File: src/fileActions.jsx [hunk 2]
```diff
@@ -546,49 +473,21 @@ export const fileActions = (path, files, selected, setSelected, clipboard, setCl
...
                 id: "paste-item",
                 title: _("Paste"),
-                onClick: () => spawnPaste(clipboard, currentPath, false),
-                isDisabled: clipboard.length === 0
-            },
-            {
-                id: "paste-as-symlink",
-                title: _("Paste as symlink"),
-                onClick: () => spawnPaste(clipboard, currentPath, true),
+                onClick: () => spawnPaste(clipboard, currentPath),
```
Comment (cockpituous) [inline src/fileActions.jsx:490]: This added line is not executed by any test.

File: src/fileActions.jsx [hunk 3]
```diff
@@ -621,7 +515,7 @@ export const fileActions = (path, files, selected, setSelected, clipboard, setCl
                     {
                         id: "paste-into-directory",
                         title: _("Paste into directory"),
-                        onClick: () => spawnPaste(clipboard, [currentPath + selected[0].name], false),
+                        onClick: () => spawnPaste(clipboard, [currentPath + selected[0].name]),
```
Comment (cockpituous) [inline src/fileActions.jsx:518]: This added line is not executed by any test.

**Comment by jelly** [comment]
> Argh, `li:nth-child(4)` tests - maybe better change that test to not assume particular indexes? (But also ok to just fix it)

Just dropping those as the test directly after it usually clicks on the same text anyway. We should have a pixel test later for the contents of the menu

**Review by martinpitt — APPROVED** [review]

**Pull request merged**